packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -120,4 +121,74 @@ test.serial('stakeAtom', async t => {
     matches(account, M.remotable('ChainAccount')),
     'account is a remotable',
   );
+
+  const atomBrand = await EV(agoricNames).lookup('brand', 'ATOM');
+  const atomAmount = AmountMath.make(atomBrand, 10n);
+
+  const res = await EV(account).delegate('cosmosvaloper1test', atomAmount);
+  t.is(res, 'Success', 'delegate returns Success');
+});
+
+test.serial('stakeAtom wallet driver', async t => {
```

Comment:
the driver is a test utility, so we're we're not testing that. 
```suggestion
test.serial('stakeAtom delegate', async t => {
```

Reply:
Meant to capture that this tested used smartWallet instead of repl. Updated this to `stakeAtom - smart wallet`, and the preceding to `stakeAtom - repl-style`.